### PR TITLE
Adapt some apparmor rules for Ubuntu

### PR DESCRIPTION
**Pull request by dodys**
#### Description:

- Add rule `all_aparmor_profiles_enforced` to Ubuntu profiles, also add SCE check and adapt bash.
- Adapt rule description and bash for `all_apparmor_profiles_in_enforced_or_complain_mode`

#### Rationale:

- This is needed for CIS

@teacup-on-rockingchair @rumch-se fyi, I've added some changes as we seen to differ on how to check for unconfined processes. Let me know if you see issues from it, even though I tried to separate per product.

### CODE REVIEW

**Comment by teacup-on-rockingchair** [comment]
Great stuff @dodys , tested it on SLE , installs the package the unconfined check works ok and warnings also :bow:

File: products/ubuntu2204/profiles/cis_level2_workstation.profile [hunk 1]
```diff
@@ -37,7 +37,8 @@ selections:
     - kernel_module_usb-storage_disabled
 
     #### 1.6.1.4 Ensure all AppArmor Profiles are enforcing (Automated)
-    #- ensure_apparmor_enforce
+    - var_apparmor_mode=enforce
```
Comment (marcusburghardt) [inline products/ubuntu2204/profiles/cis_level2_workstation.profile:40]: This variable is not used by `all_apparmor_profiles_enforced` rule. It is intentional to define it here?

Comment (marcusburghardt) [reply products/ubuntu2204/profiles/cis_level2_workstation.profile:40]: Thanks @dodys

**Review by marcusburghardt — APPROVED** [review]

**Pull request merged**